records_mover/records/job/mover.py

```diff
@@ -35,6 +35,6 @@ def run_records_mover_job(source_method_name: str,
         source = source_method(**source_kwargs)
         target = target_method(**target_kwargs)
         return records.move(source, target, processing_instructions)
-    except Exception as e:
-        logger.error(e)
+    except Exception:
+        logger.exception('Error while processing')
```

Comment:
I added a typestub for logging, and found some cases where we were handing an exception object as the first argument to e.g. logger.info().  When tested, that outputs an empty line, not any exception information, so let's try something better!